Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM python:3.11
 
+RUN pip install poetry
+
 WORKDIR /app
-COPY scripts/ /app
 
-RUN pip install -r requirements.txt
+COPY poetry.lock /app
+COPY pyproject.toml /app
+RUN poetry install
+
+COPY scripts/ /app
```

Comment:
You could just use one COPY statement and save some layers.

Reply:
Thanks, good idea. I added the 2 files to 1 COPY statement.